modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -99,7 +99,7 @@ if (CONFIG_BUILD_WITH_TFM)
     set(TFM_PARTITION_LOG_LEVEL "TFM_PARTITION_LOG_LEVEL_INFO")
   elseif (CONFIG_TFM_PARTITION_LOG_LEVEL_ERROR)
     set(TFM_PARTITION_LOG_LEVEL "TFM_PARTITION_LOG_LEVEL_ERROR")
-  elseif (CONFIG_TFM_PARTITION_LOG_LEVEL_SILENCE)
+  elseif (CONFIG_TFM_PARTITION_LOG_LEVEL_SILENCE OR CONFIG_TFM_LOG_LEVEL_SILENCE)
```

Comment:
This means we might have a mismatch between the Kconfig and the resulting TF-M config, right?

Is there no way around this? This is unfortunate.